e2e/cypress/integration/files_and_attachments/image_link_preview_spec.js

```diff
@@ -132,4 +132,114 @@ describe('Image Link Preview', () => {
         // # Close the image preview modal
         cy.get('body').type('{esc}');
     });
+
+    it('MM-T1447 Images below a min-width and min-height are posted in a container that is clickable', () => {
+        const listOfMinWidthHeightImages = [
+            {
+                filename: 'image-20x20.jpg',
+                originalSize: {width: 20, height: 20},
+                thumbnailSize: {width: 20, height: 20},
+                containerSize: {height: 46},
+            },
+            {
+                filename: 'image-50x50.jpg',
+                originalSize: {width: 50, height: 50},
+                thumbnailSize: {width: 50, height: 50},
+            },
+            {
+                filename: 'image-60x60.jpg',
+                originalSize: {width: 60, height: 60},
+                thumbnailSize: {width: 60, height: 60},
+            },
+            {
+                filename: 'image-400x400.jpg',
+                originalSize: {width: 400, height: 400},
+                thumbnailSize: {width: 350, height: 350},
+            },
+            {
+                filename: 'image-40x400.jpg',
+                originalSize: {width: 40, height: 400},
+                thumbnailSize: {width: 35, height: 350},
+                containerSize: {width: 46},
+            },
+            {
+                filename: 'image-400x40.jpg',
+                originalSize: {width: 400, height: 40},
+                thumbnailSize: {width: 400, height: 40},
+                containerSize: {height: 46},
+            },
+            {
+                filename: 'image-1000x40.jpg',
+                originalSize: {width: 1000, height: 40},
+                thumbnailSize: {width: 971, height: 38.82},
+                containerSize: {height: 46},
+            },
+            {
+                filename: 'image-1600x40.jpg',
+                originalSize: {width: 1600, height: 40},
+                thumbnailSize: {width: 971, height: 24.26},
+                previewSize: {width: 1248, height: 31.18},
+                containerSize: {height: 46},
+            },
+        ];
+
+        listOfMinWidthHeightImages.forEach((imageWithMinWidthHeight) => {
+            // # Upload Image as attachment and post it
+            cy.get('#fileUploadInput').attachFile(imageWithMinWidthHeight.filename);
+            cy.postMessage(`${Date.now()}-${imageWithMinWidthHeight.filename}`);
```

Comment:
nit: I'm good as is but since we're now testing in isolation, `Date.now()` can be safely removed.